modules/snooker/src/test/ts/browser/resize/ColumnSizesTest.ts

```diff
@@ -40,7 +43,12 @@ UnitTest.test('ColumnSizes.getPixelWidths', () => {
   sTest('Pixel Table - Column widths with missing widths on some cells should be the raw size of the cell', pixelTableMissingWidthsHtml, () => [ 200, 200 ]);
   sTest('Pixel Table - Column width should be the size of the table when using colspans', tableWithSpansHtml, () => [ 0, 400 ]);
   sTest('None Table - Column widths should be the computed size of the cell', noneTableHtml, (width) => [ width, width ]);
-  sTest('None Table - Column widths for cols should be the computed size of the cell', noneTableWithColsHtml, (width) => [ width + 2, width + 2 ]); // Add 2 to account for the borders
+  if (isChrome92) {
+    // Handle bug in Chrome 92
```

Comment:
Can we please link to a JIRA here so we have more context, as this provides very little about why we are doing this (maybe just create a placeholder for now?